.github/workflows/test-and-upload-coverage.yml

```diff
@@ -143,6 +171,20 @@ jobs:
           make deps:modules
           make deps:test
 
+      # We have to checkout the source-hub repo and install it ourselves because it
+      # contains replace commands in its go.mod file.
+      - name: Checkout sourcehub code into the directory
+        if: ${{ matrix.acp-type == 'source-hub' }}
+        uses: actions/checkout@v3
```

Comment:
todo: This is now deprecated. Please use the version similar to other `actions/checkout`

Reply:
Nice catch, this was added a month or so ago, before getting blocked in the last sprint I think :)

- [ ] Bump checkout version